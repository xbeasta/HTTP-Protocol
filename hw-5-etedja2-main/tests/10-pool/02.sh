#!/bin/bash

# check that Test 9 and 10 enabled
# if this fails, print error and exit
STR="Please provide the number of threads to use\nTest 9 and 10 enabled"
if ./hw5  | grep -q "$STR"; then
    echo "ERROR: Test 9 and 10 not enabled"
    exit 1
fi

# check tests 05-08
grep "TEST 05-file PASSING" results
grep "TEST 06-ping-echo-write-read-file PASSING" results
grep "TEST 07-write-write-write PASSING" results
grep "TEST 08-stats PASSING" results