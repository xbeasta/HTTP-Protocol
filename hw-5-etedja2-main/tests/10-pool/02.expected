[42m                   TEST 05-file PASSING                [0m
[42m                   TEST 06-ping-echo-write-read-file PASSING                [0m
[42m                   TEST 07-write-write-write PASSING                [0m
[42m                   TEST 08-stats PASSING                [0m
